src/modules/app/browser-app.module.ts

```diff
@@ -54,12 +54,13 @@ export function getRequest(transferState: TransferState): any {
     // forRoot ensures the providers are only created once
     IdlePreloadModule.forRoot(),
     RouterModule.forRoot([], {
-      // enableTracing: true,
-      useHash: false,
-      scrollPositionRestoration: 'enabled',
-      anchorScrolling: 'enabled',
-      preloadingStrategy: NoPreloading
-    }),
+    // enableTracing: true,
+    useHash: false,
+    scrollPositionRestoration: 'enabled',
+    anchorScrolling: 'enabled',
+    preloadingStrategy: NoPreloading,
+    relativeLinkResolution: 'legacy'
```

Comment:
Did you check to see whether `relativeLinkResolution: 'legacy'` is needed? At first sight I couldn't find any instances of [the bug it's trying to stay backwards compatible with](https://angular.io/api/router/ExtraOptions#relativeLinkResolution)

Reply:
@artlowel : this was an *automated change*. It looks like it was automatically added when I ran `npx @angular/cli@11 update @angular/core@11 @angular/cli@11 --force`.  If we wanted to try to remove it, we could do so (either in this PR or in a later PR). I avoided making any manual changes in this PR except where necessary.

My assumption though is that the `update` script must have seen something in our code that implied it needed to enable `legacy` behavior.  Either that, or it's enabled by default in every Angular 11 upgrade.

Reply:
@artlowel @tdonohue 

for what says the documentation here https://v11.angular.io/api/router/ExtraOptions#relativeLinkResolution if we don't use the legacy option we should check if all relative links work properly

Reply:
As discussed in today's meeting, I'll try and remove this `relativeLinkResolution: 'legacy'` option today and retest. If it works fine, I'll push it up into this PR & get a final review before we merge this (as we agreed we want to merge this soon). If it doesn't work (and I cannot easily figure out a solution), I'll log a separate ticket to remove the `relativeLinkResolution: 'legacy'` & we'll move this PR forward as-is.

Reply:
@artlowel and @atarix83 : I removed `relativeLinkResolution: 'legacy'` and manually checked our links to make sure we didn't need it (I couldn't find anything that required legacy behavior).  After removing it, I retested everything using `yarn start` and I'm seeing no change in behavior.  

Could you retest?  That way we can get this merged as soon as reasonably possible.